### PR TITLE
Value should reflect user changes in inputs

### PR DESCRIPTION
**Pull request by ellenaua**
Refs https://github.com/PolymerElements/gold-cc-expiration-input/issues/53

### CODE REVIEW

**Comment by ellenaua** [comment]
refs https://github.com/PolymerElements/gold-cc-expiration-input/issues/53

File: gold-cc-expiration-input.html [hunk 1]
```diff
@@ -148,9 +148,16 @@
...
-            this.value = event.detail.month + '/' + event.detail.year;
-          }
+          var month = event.detail.month || '';
+          var year = event.detail.year || '';
+          this.value = this._buildValue(month, year);
+        },
+
+        _buildValue: function(month, year){
+          if (!month && !year) return '';
+          // @TODO when only month is entered, return month without '/'
```
Comment (notwaldorf) [inline gold-cc-expiration-input.html]: Will you be fixing this TODO? We don't tend to merge PRs with TODOs in them :)

**Comment by notwaldorf** [comment]
@ellenaua this looks great! I just left one comment before I can actually merge it :)

**Comment by notwaldorf** [comment]
Going to close this PR since it's been a month without answer. Please re-open it and @mention me when it's ready for another look!

**Pull request closed without merging**